extensions/blocks/story/player/use-resize-observer.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createElement,
+	useCallback,
+	useEffect,
+	useMemo,
+	useRef,
+	useState,
+} from '@wordpress/element';
+import { useResizeObserver as useResizeObserverWordpress } from '@wordpress/compose';
```

Comment:
Since `useResizeObserver` is not available in WordPress 5.4, you won't be able to rely on it without a check for it first, or you'll break things on sites that run WP 5.4.

Reply:
Yep the check happens on the next line actually

Reply:
🤦 That will teach me to comment without paying attention. Thank you!